### PR TITLE
Update TLS wording to be stronger

### PR DESCRIPTION
**Pull request by klarose**
We want to make TLS a requirement, not a suggestion, so I've chagned
the wording around TLS to reflect this. I also added a section to the
security section discussing the motivation behind the requirement.

### CODE REVIEW

File: draft-ietf-capport-architecture.xml [hunk 1]
```diff
@@ -601,10 +601,18 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
...
-         trust the API server, server authentication will be utilized using TLS
+         trust the API server, server authentication will be performed using TLS
          mechanisms.
        </t>
      </section>
+     <section title="Secure APIs">
+       <t>
+           The solution described here requires that the API be secured using TLS.
+           This requirement follows from the fact that the user equipment may provide
+           secrets to the API. The API must ensure the integrity of this information,
```
Comment (martinthomson) [inline draft-ietf-capport-architecture.xml]: The current api includes secrets from the api, not the other way around.

Comment (klarose) [reply draft-ietf-capport-architecture.xml]: Thanks. I'll rephrase it to be more generic.

Comment (tfpauly) [reply draft-ietf-capport-architecture.xml]: Yes, perhaps something like:

"This is required to allow the user equipment and API server to exchange secrets which can be used to validate future interactions."

Comment (klarose) [reply draft-ietf-capport-architecture.xml]: Thanks, Tommy. I'll use that wording. :)
